src/Component/Authentication/Forgot.jsx

```diff
@@ -0,0 +1,264 @@
+import React, { useState } from "react";
+import Alert from "@mui/material/Alert";
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
+import LoginIcon from "@mui/icons-material/Login";
+import {
+  Container,
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  Typography,
+  SvgIcon,
+} from "@mui/material";
+import Box from "@mui/material/Box";
+import CloseIcon from "@mui/icons-material/Close";
+import IconButton from "@mui/material/IconButton";
+import Paper from "@mui/material/Paper";
+import Chip from "@mui/material/Chip";
+import Link from "@mui/material/Link";
+import { TypeSpecimenOutlined } from "@mui/icons-material";
+
+const isEmail = (email) =>
+  /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(email);
+
+function Forgot() {
+  const [emailInput, setEmailInput] = useState();
```

Comment:
_"Warning: A component is changing an uncontrolled input to be controlled. This is likely caused by the value changing from undefined to a defined value, which should not happen. Decide between using a controlled or uncontrolled input element for the lifetime of the component."_

This error occurs when I input an email address. To avoid the 'uncontrolled to controlled input' warning, we can initialize the state variable with an empty string: useState('')

Reply:
Changed as per your comment to useState('').Thanks @nihalerdal